src/lingva/extract.py

```diff
@@ -151,16 +151,15 @@ def create_catalog(width, copyright_holder, package_name, package_version, msgid
     catalog.package_name = package_name
     catalog.metadata_is_fuzzy = True
     catalog.metadata = OrderedDict()
-    catalog.metadata["Project-Id-Version"] = " ".join(
-        filter(None, [package_name, package_version])
-    )
+    catalog.metadata["Project-Id-Version"] = f"{package_name} {package_version}"
```

Comment:
Might break something for someone

Reply:
In fact, these parameters shouldn't be empty: currently they have [default values](https://github.com/vacanza/lingva/blob/512ac28552ef09565c4f2170b02f9fe3cb0b4a3f/src/lingva/extract.py#L346) in `main`, but if someone called `save_catalog()` directly...